content/guides/dashboard/github-integration.md

```diff
@@ -283,6 +283,11 @@ steps:
 3. Click **Configure** beside the Cypress app.
 4. Click **Uninstall** below the "Uninstall Cypress" section.
 
+## Troubleshooting
+
+See
+[Common Problems and Solutions](/guides/continuous-integration/github-actions#Common-problems-and-solutions).
```

Comment:
My bad on the URL, seems like casing matters here. Always good to make sure these go to the intended location in the preview URL. This should anchor to the right spot:

`/guides/continuous-integration/github-actions#Cypress-Real-World-Example-with-GitHub-Actions`